lib/auth/touchid/api_test.go

```diff
@@ -952,3 +952,8 @@ func (u *fakeUser) WebAuthnIcon() string {
 func (u *fakeUser) WebAuthnName() string {
 	return u.name
 }
+
+// Artificially added test method
+func TestNew(t *testing.T) {
```

Comment:
Do we need those?

Reply:
Yes, we need them for now. I will remove those before merge.